pages/4_AWS_Health.py

```
@@ -43,9 +68,20 @@
-        data=filtered_df.to_csv(index=False),
-        file_name="aws_events_filtered.csv",
-        mime="text/csv",
-    )
\ No newline at end of file
+        label=f"Download Filtered Data as {download_format}",
+        data=data,
+        file_name=f"aws_events_filtered.{download_format.lower()}",
+        mime=mime_type,
+    )
+
+sender_email = st.text_input("Sender Email (Verified in SES)")
```

Ruff (0.8.2)

`st` may be undefined, or defined from star imports

(F405)